src/mod/ME_Collapse.c

```diff
@@ -100,241 +101,250 @@ MVertex_ptr ME_Collapse(MEdge_ptr e, MVertex_ptr vkeep_in, int topoflag,
         vdel_external = 1;
         break;
       }
-    }                                          
+    }
     
     List_Delete(vedges);
-
+    
     /* check if any face connected to vdel has only one region
-       connected to it */
-
+     connected to it */
+    
     if (!vdel_external) {
       vfaces = MV_Faces(vdel);
-
+      
       idx1 = 0;
       while ((face = (MFace_ptr) List_Next_Entry(vfaces,&idx1))) {
         List_ptr fregs = MF_Regions(face);
         int nfr = fregs ? List_Num_Entries(fregs) : 0;
         if (fregs) List_Delete(fregs);
-
+        
         if (nfr == 1) {
           vdel_external = 0;
           break;
         }
       }
-
+      
       List_Delete(vfaces);
     }
-
+    
     if (vdel_external) {
       /* swap the vertices in the hope that vkeep is not also on an
-         external boundary. Since we have to go through with the
-         collapse anyway, there is no use of doing a detailed check
-         for whether vkeep is also on an external boundary */
-
+       external boundary. Since we have to go through with the
+       collapse anyway, there is no use of doing a detailed check
+       for whether vkeep is also on an external boundary */
+      
       MVertex_ptr vtemp = vdel;
       vdel = vkeep;
       vkeep = vtemp;
     }
     
   }
-
-
+  
+  
   if (status == 0)
-    return NULL;   /* Cannot collapse due to constraints of topological
-		   conformity with geometric model */
-
+  return NULL;   /* Cannot collapse due to constraints of topological
+                  conformity with geometric model */
+  
   *deleted_entities = List_New(10);
-
+  
   /* Need to collect this in advance because the info gets messed up later */
-
+  
   efaces = ME_Faces(e);
   eregs = ME_Regions(e);
-
-
-  /* Replace vdel with vkeep in all edges connected to vdel */
```

Comment:
I am finding it a bit hard to figure out how this changed. Did the steps get reordered?